### PR TITLE
Add manual restore process

### PR DESCRIPTION
**Pull request by Kidswiss**
## Checklist

- [X] Try to isolate changes into separate PRs (to build a better changelog).
- [X] Categorize the PR by setting a good title and adding one of the labels:
      `change`, `decision`, `requirement/quality`, `requirement/functional`, `dependency`
      as they show up in the changelog
- [X] Link this PR to related issues if applicable.

### CODE REVIEW

File: docs/modules/ROOT/pages/runbooks/exoscale/restore_dbaas.adoc [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+# Create a fork from backup
+# With point-in-time
+exo dbaas create pg startup-4 fork-test-pg --pg-recovery-backup-time 2021-10-30 23:35 --pg-fork-from $myinstance -z ch-gva-2
+
+# Without point-in-time (restores latest available)
+exo dbaas create pg startup-4 fork-test-pg --pg-fork-from $myinstance -z ch-gva-2
+----
+
+There will be a new instance, which is not AppCat managed!
```
Comment (tobru) [inline docs/modules/ROOT/pages/runbooks/exoscale/restore_dbaas.adoc:38]: Crossplane should be able to adopt an instance by setting the `externalName` (if I remember correctly). That would be much easier than doing another dump/restore dance.

**Review by tobru — CHANGES_REQUESTED** [review]
We should find a way to let the user restore backups in self-service somehow.

**Comment by Kidswiss** [comment]
> We should find a way to let the user restore backups in self-service somehow.

Yes of course, but the manual restore will come in handy sooner or later for sure.

File: docs/modules/ROOT/pages/runbooks/exoscale/restore_dbaas.adoc (hunk 1 above)
Comment (Kidswiss) [reply docs/modules/ROOT/pages/runbooks/exoscale/restore_dbaas.adoc:38]: Good point, but haven't tried that yet...

File: docs/modules/ROOT/pages/runbooks/exoscale/restore_dbaas.adoc [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+NOTE: You can't restore from a `+hobbyist-2+` instance. You have to scale it up first.
+
+This documentation is for manually restoring a DbaaS instance.
+
+== Scale up Instance
+
+[source,shell]
+----
+exo dbaas update -z ch-gva-2 $myinstance --plan startup-4
```
[the remark this answers is not in the document]
Comment (Kidswiss) [reply docs/modules/ROOT/pages/runbooks/exoscale/restore_dbaas.adoc:13]: Will add the info.

File: docs/modules/ROOT/pages/runbooks/exoscale/restore_dbaas.adoc (hunk 1 above)
Comment (tobru) [reply docs/modules/ROOT/pages/runbooks/exoscale/restore_dbaas.adoc:38]: Maybe something to plan for? Would make restoring much easier IMHO...

Comment (Kidswiss) [reply docs/modules/ROOT/pages/runbooks/exoscale/restore_dbaas.adoc:38]: I agree, but for the time being, if shit hits the fan we need to do the dump dance to do restores.

We can rework this page again once we have implemented either the `external-name` or the ability to do the restore via claim.

Comment (Kidswiss) [reply docs/modules/ROOT/pages/runbooks/exoscale/restore_dbaas.adoc:71]: Created a tech-debt: https://github.com/vshn/provider-exoscale/issues/78

File: docs/modules/ROOT/pages/runbooks/exoscale/restore_dbaas.adoc [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+# Without point-in-time (restores latest available)
+exo dbaas create pg startup-4 fork-test-pg --pg-fork-from $myinstance -z ch-gva-2
+----
+
+There will be a new instance, which is not AppCat managed!
+To get the data into an AppCat Managed Service again:
+
+* dump the data from the restore
+* restore the dump either into the existing AppCat service, or a new one
```
Comment (tobru) [inline docs/modules/ROOT/pages/runbooks/exoscale/restore_dbaas.adoc:75]: Your team colleagues will certainly be happy if you document these steps in detail.

**Review by tobru — CHANGES_REQUESTED** [review]

**Review by tobru — APPROVED** [review]

**Pull request merged**